packages/components/package.json

```diff
@@ -50,7 +50,7 @@
   },
   "homepage": "https://github.com/LabKey/labkey-ui-components#readme",
   "dependencies": {
-    "@labkey/api": "1.30.0",
+    "@labkey/api": "1.30.1-crossFolderPermMessage.0",
```

Comment:
is there a PR for the labkey-api-js changes?

Reply:
There is now. Thanks!